src/lib/pages/backoffice/Document/DocumentDetails/DocumentPurchaseOrders/DocumentPurchaseOrders.js

```diff
@@ -0,0 +1,148 @@
+
+import { dateFormatter } from '@api/date';
+import React, { Component } from 'react';
+import PropTypes from 'prop-types';
+import { withCancel } from '@api/utils';
+import { SeeAllButton } from '@components/backoffice/buttons/SeeAllButton';
+import { orderApi } from '@api/acquisition';
+import { ResultsTable } from '@components/ResultsTable/ResultsTable';
+import { Loader } from '@components/Loader';
+import { Link } from 'react-router-dom';
+import { AcquisitionRoutes } from '@routes/urls';
+import { Message } from 'semantic-ui-react';
+
+export default class DocumentPurchaseOrders extends Component {
+
+  constructor(props) {
+    super(props);
+    this.state = {
+      orders: {},
+      isLoading: true,
+      error: false
```

Comment:
Very minor: just to keep it consistent with the rest of the code:
```suggestion
      hasError: false
```